tests/resources/aria-at-harness.mjs

```diff
@@ -623,13 +648,32 @@ function submitResult(event) {
     summary
   };
 
+  let data = {
+    test: document.title,
+    details: behaviorResults,
+    status: overallStatus
+  };
+
+  appendJSONResults(data);
+
+  // send message to parent if test is loaded in iFrame
+  if (window.parent && window.parent.postMessage) {
+    window.parent.postMessage({
+      type: 'results',
+      data: data
+    }, '*');
+  }
+
   endTest();
```

Comment:
You call end test, but reviewed the function?

Reply:
the original `endTest` became three separate functions: `showResultsTable` (handles replacing HTML), `endTest` (handles closing test popup window), and `appendJSONResults` (handles inserting JSON data into script element). (The earlier bug was that `appendJSONResults` was called before  `showResultsTable` which replaced the documents entire html! 🤦)